robotd/devices.py

```diff
@@ -227,10 +227,16 @@ def start(self):
         self.vision_thread = Thread(target=self._vision_thread)
         self.vision_thread.start()
 
+    @staticmethod
+    def _serialise_marker(marker : Token):
+        d = marker.__dict__
+        d['homography_matrix'] = [[c for c in r] for r in marker.homography_matrix]
```

Comment:
Assuming the aim of these is to perform a copy? In which case the intent would be clearer using `copy.deepcopy`.

Reply:
The aim of these is to convert from a numpy ndarray to a simple list of lists.

Reply:
Though now that I think about it, it might be more obvious to change it to 
`[list(r) for r in marker.homography_matrix]

Reply:
Looks like there's a builtin function for this, can we use it? https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.tolist.html#numpy.ndarray.tolist